### ISSUE TITLE
File - Save a Copy... closes document being viewed after saving.

### ISSUE THREAD

**Issue by manglewit** [linuxmint/xreader#629]
### Distribution

Linux Mint 21.2

### Package version

3.8.2

### Frequency

Always

### Bug description

Open a PDF file. Choose File -- Save a Copy... and save a copy. On completion of the save, xreader exits. This is not the wanted behaviour.

### Steps to reproduce

Create a PDF file e.g. by exporting from LibreOffice  (I used LibreOffice 7.3.7.2 Writer. Open newly created PDF file in xreader. Choose File -- Save a Copy... and save a copy. On completion of the save, xreader exits. This is not the wanted behaviour.

### Expected behavior

I expect a copy of the PDF currently being viewed to be saved as requested in teh Save dialogue, and once save for the session viewing the file to continue without exiting.

### Additional information

This appears to be a reoccurrence of a bug that was apparently fixed.

https://github.com/linuxmint/xreader/issues/444

**Comment by mtwebster** [comment]
Fixed. Duplicate of https://github.com/linuxmint/xreader/issues/609

**Comment by manglewit** [comment]
I thought this might be different as it wasn't an annotated PDF (#609), just an 'ordinary' one.

This is where I show my ignorance - are we waiting on upstream, or some other process before an update can be sent out via the Update Manager (in other words, the repository gets updated with a fixed version)? Point me towards the Fine Manual if the process is documented somewhere so I can read and set my expectations accordingly.
